shim/tests/mod.rs

```diff
@@ -23,47 +23,9 @@ fn test_no_arguments() {
         .append_context("turbo", "no arguments")
         .append_context(
             "expect",
-            "`turbo` with no arguments should exit with code 1",
+            "`turbo` with no arguments should exit with code 2",
         )
-        .code(1);
-}
-
-#[test]
-fn test_find_turbo_in_example() {
```

Comment:
Deleted because example tests cover this and we'd need to actually initialize the example here.